src/api/app/views/webui2/webui/request/_decision_tab.html.haml

```diff
@@ -16,12 +16,12 @@
           Forward submit request to
           #{project_or_package_link(project: forward[:project], package: forward[:package], short: true)}
     %hr
-  - if state.in?(['new', 'review']) && is_target_maintainer
-    - confirmation = (state == 'review' ? { confirm: 'Do you really want to approve this request, despite of open review requests?' } : {})
-    = submit_tag 'Accept request', name: 'accepted', class: 'btn btn-primary mr-2', data: confirmation
     - unless is_author
       = submit_tag 'Decline request', name: 'declined', class: 'btn btn-danger mr-2'
   - if is_author && state.in?(['new', 'review', 'declined'])
     = submit_tag 'Revoke request', name: 'revoked', class: 'btn btn-danger mr-2'
   - if state == 'declined'
     = submit_tag 'Reopen request', name: 'new', class: 'btn btn-warning'
+  - if state.in?(['new', 'review']) && is_target_maintainer
```

Comment:
Shouldn't the conditions of this `if` be duplicated in line 19?

Reply:
Oups, you are right. Fixed!